hardware_interface/include/fake_components/generic_system.hpp

```diff
@@ -97,7 +97,8 @@ class GenericSystem : public hardware_interface::BaseInterface<hardware_interfac
   bool get_interface(
     const std::string & name,
     const std::vector<std::string> & interface_list,
-    const std::string & interface_name, const uint vector_index,
+    const std::string & interface_name,
+    const size_t vector_index,
```

Comment:
@Karsten1987 OK, but size_t is even less logical IMO. But it seems it is used all around standard library so OK. It makes kind of sense when working with `size()` method.